src/bentoml/_internal/bento/bento.py

```diff
@@ -330,7 +324,7 @@ def export(
         subpath: t.Optional[str] = None,
     ) -> str:
         # copy the models to fs
-        models_dir = self._fs.makedir("models", recreate=True)
+        models_dir = self.fs.makedir("models", recreate=True)
```

Comment:
I'm not sure we should expose `fs` itself. Maybe we can write some abstractions?

Reply:
I think fs is already itself an abstraction? I think we should expose this fs object.

Reply:
I'd prefer not to expose external APIs we have no control over if we have a choice, is why. It's also something that would need to be replaced in the potentially soon:tm: rust rewrite.

Reply:
i see. makes sense then